packages/components/src/components/popover/popover.tsx

```diff
@@ -52,47 +55,65 @@ export class Popover {
   private popover: HTMLDivElement;
   private button: HTMLButtonElement;
 
+  private isNative: boolean = false;
+  private table: HTMLElement;
```

Comment:
Perhaps rename to `parentTable`, `parentTableEl` or `parentTableElement` for more clarity. Currently it is named just like the child elements `popover` and `button`.